core/camera/src/main/java/com/google/jetpackcamera/core/camera/CameraUseCase.kt

```diff
@@ -67,7 +67,11 @@ interface CameraUseCase {
         ignoreUri: Boolean = false
     ): ImageCapture.OutputFileResults
 
-    suspend fun startVideoRecording(onVideoRecord: (OnVideoRecordEvent) -> Unit)
+    suspend fun startVideoRecording(
+        videoCaptureUri: Uri?,
+        ignoreUri: Boolean = false,
```

Comment:
nit: every reference to ignoreUri is `!ignoreUri`. Feels a little weird because i read it in my head like "not not this".
Would it be better to just name it something that starts positive like`shouldUseUri`and adjust the references appropriately?

Reply:
makes sense. Renamed and reversed the checks